README.md

```diff
@@ -23,14 +23,16 @@ $ npm install -g git+https://git@github.com/ConsenSys/truffle-security.git
 
 ## Configuration
 
-Currently, the plugin must be activated on a per-project basis. Add the following to `truffle.js` in the root directory of your Truffle project:
+Currently, the plugin must be activated on a per-project basis. Add the following to `truffle-config.js` in the root directory of your Truffle project:
```

Comment:
Should we also mention that we automatically register the plugin for the user if they install it from the project root here?

Reply:
Done.